apps/src/javaide/JavaConsole.jsx

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import PropTypes from 'prop-types';
+import {appendInputLog} from './redux';
```

Comment:
We've started to run into problems because we have so many files named "redux.js" - javaideRedux.js may be better here.